rs/nns/governance/canister/governance.did

```diff
@@ -502,6 +512,9 @@ type ManageNeuronCommandRequest = variant {
   StakeMaturity : StakeMaturity;
   MergeMaturity : MergeMaturity;
   Disburse : Disburse;
+  RefreshVotingPower : RefreshVotingPower;
```

Comment:
This is the root of this change. All other changes in this PR can be deduced from this (via failing tests).